minhton/include/minhton/algorithms/find_end/minhton_find_end_algorithm.h

```diff
@@ -14,6 +14,7 @@
 #include "minhton/message/types_all.h"
 
 namespace minhton {
+#define MAX_HOP_COUNT 64U
```

Comment:
Could this become a global static (and maybe even constexpr) variable instead of a macro?
This is currently only used in a single .cpp file. Could this also be moved to the file to limit the visibility. Or if this should be used by multiple files in the future: Move this to ``constants.h`` to have all configuration variables in a single place.